.github/workflows/pants.yaml

```
@@ -18,61 +18,21 @@ jobs:
   build:
     name: Perform CI Checks
     needs: org-check
-    env:
-      PANTS_CONFIG_FILES: pants.ci.toml
     runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: [3.7]
     steps:
     - uses: actions/checkout@v2
-    - name: Get Pants version
-      id: pants_version
-      run: |
-        # Capture the "pants_version = " line from config.
-        PANTS_VERSION=$(grep -E '^pants_version\s*=' pants.toml)
-        echo "::set-output name=pants_version::$PANTS_VERSION"
-    - uses: actions/cache@v2
-      id: cache_pants_setup
-      with:
-        path: |
-          ~/.cache/pants/setup
-        key: pants-setup-${{ steps.pants_version.outputs.pants_version }}
-    - uses: actions/cache@v2
-      id: cached_named_caches
+    - uses: st2sandbox/init-pants-action@main
```

```suggestion
    - uses: pantsbuild/init-pants-action@v1
```
This PR demonstrates this new action which I would like to transfer to the pantsbuild org.
Once transferred, we can make a tag+release and publish the action to the GitHub marketplace.